weldx/core.py

```diff
@@ -778,9 +776,6 @@ def __post_init__(self):
         if not isinstance(self.values, (pint.Quantity, xr.DataArray)):
             self.values = Q_(self.values)
 
-        if self.symbol is not None and len(self.symbol) > 1:
-            raise ValueError(f"Cannot use symbol {self.symbol}")
-
```

Comment:
To my knowledge, this is true if you use a string expression. But you can use `sympy.symbols` to create symbol names of arbitrary length and add them to an expression or create a new one. I used this for the translation function of the shape segment. There we do not know, which parameter names will be already in use and there might also be more than one translation. So I just named the translation parameters 'translation_' + number. Of course, this triggered the exception, so I removed it. Also, the safeguard should not work as expected since `"an * expression"` will create single character symbols anyway, right?

Reply:
Since `SeriesParameter` is just a container to represent parameters/coordiantes/symbols there is no `sympy` involved here.

But apparently it does work with multi character strings anyway? So it's not really needed
```python
from weldx import Q_, GenericSeries

GenericSeries(
    "a*time + b",
    parameters=dict(a=Q_([3, 0, 0], "mm/s"), b=Q_([0, 2, 2], "cm")),
    units=dict(time="s"),
)
#> <GenericSeries>
#> Expression:
#> 	a*time + b
#> Parameters:
#> 	a = [3 0 0] mm / s
#> 	b = [0 2 2] cm
#> Free Dimensions:
#> 	time in s
#> Other Dimensions:
#> 	['dim_0']
#> Units:
#> 	mm
#> 
```